crates/cxx-qt-build/src/opts.rs

```diff
@@ -42,3 +47,40 @@ impl CxxQtBuildersOpts {
         self
     }
 }
+
+/// Options for qobject_headers
+///
+/// QObjectHeaderOpts can be created using the `From<impl AsRef<Path>>` trait.
+/// ```
```

Comment:
should this be `rust` so that it is tested or `rust,ignore` if not ?

Reply:
Doc comments are `rust` by default.
They are already tested, as cargo check errored out on them when I forgot to add the `use` statements :)